src/jvm/test/resources/expected_logs/multiple_suspension_points_channels.txt

```diff
@@ -10,116 +10,116 @@ The number next to an exception name helps you find its stack trace provided aft
-|                                                                                                                                                        |   counter ➜ 5 at MultipleSuspensionPointsChannelsTraceRepresentationTest.operation2(MultipleSuspensionsTests.kt:170)                                   |
-|                                                                                                                                                        |   result: IllegalStateException #2                                                                                                                     |
-| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                       Thread 1                                                                        |                                                                       Thread 2                                                                        |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|                                                                                                                                                       | operation2(): IllegalStateException #2                                                                                                                |
+|                                                                                                                                                       |   channel1.receive(): 1 at MultipleSuspensionPointsChannelsTraceRepresentationTest.operation2(MultipleSuspensionsTests.kt:166)                        |
+|                                                                                                                                                       |     BufferedChannel.receive$suspendImpl(BufferedChannel#1): 1 at BufferedChannel.receive(BufferedChannel.kt:0)                                        |
```

Comment:
Can we get rid of `$suspendImpl` function calls?

Reply:
Done